docs/two-phase-commit-transactions.md

```diff
@@ -1,33 +1,27 @@
-# Two-phase Commit Transactions
+# Two-Phase Commit Transactions
 
-ScalarDB also supports two-phase commit style transactions called *Two-phase Commit Transactions*.
-With Two-phase Commit Transactions, you can execute a transaction that spans multiple processes/applications (e.g., Microservices).
+ScalarDB supports executing transactions by using a two-phase interface called *two-phase commit transactions*. With two-phase commit transactions, you can execute a transaction that spans multiple processes or applications, like in a microservice architecture.
 
-This document briefly explains how to execute Two-phase Commit Transactions in ScalarDB.
+This page explains how two-phase commit transactions work in ScalarDB and how to configure and execute them in ScalarDB.
 
-## Overview
+## How two-phase commit transactions work in ScalarDB
 
-ScalarDB normally executes transactions in a single transaction manager instance with a one-phase commit interface, which we call normal transactions.
-In that case, you begin a transaction, execute CRUD operations, and commit the transaction in the same transaction manager instance.
+ScalarDB normally executes transactions in a single transaction manager instance with a one-phase commit interface, which we call normal transactions. In normal transactions, you begin a transaction, execute CRUD operations, and commit the transaction in the same transaction manager instance.
```

Comment:
@brfrn169 (cc: @feeblefakie) I can see how the phrase `normal transactions` could be a little awkward. How about we use the following instead?:

```suggestion
ScalarDB normally executes transactions in a single transaction manager instance by using a one-phase commit interface. In one-phase commit transactions, you begin a transaction, execute CRUD operations, and commit the transaction in the same transaction manager instance.
```

Reply:
Sounds good to me.

@feeblefakie What do you think?

Reply:
I think `one-phase commit transactions` sounds like optimized two-phase commit transactions, so it might be misleading.
So, how about `transactions with one-phase commit interface`?

Reply:
@feeblefakie I see. I'm OK with your suggestion👍

```suggestion
ScalarDB normally executes transactions in a single transaction manager instance with a one-phase commit interface. In transactions with a one-phase commit interface, you begin a transaction, execute CRUD operations, and commit the transaction in the same transaction manager instance.
```